ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java

```diff
@@ -168,4 +170,22 @@ public String getElementText(By locator) {
 		SelenideElement element = this.getRootElement().find(locator);
 		return element.shouldBe(visible).getText();
 	}
+
+	public void checkButtonStatus(String buttonTitle, String status) {
+		log.info("checking button {} status", buttonTitle);
+		Condition condition;
+		switch(status){
+			case "Active":
+			case "Visible":
+				condition = visible;
+				break;
+			case "Disabled":
+			case "Inactive":
+				condition = disabled;
+			default:
+				condition = visible;
+				break;
+		}
+		SelenideElement buttonElement = this.getButton(buttonTitle).shouldBe(condition);
```

Comment:
Is there any benefit to have such a method. Imho you always can use getButton() and inject the condition you need in a appropriate code.

Reply:
no problem. method removed.

Reply:
just to be sure what you mean. 
st. like "..shouldBe(Condition.attribute(status))"?
(where 'status' is String value from cucumber step)

Reply:
I haven't realized that you consume status from step also. In that case it's ok. But maybe I would refactor it to more generic approach, like `conditionValueOf(String status)` that return selenide Conditions based on provided string to some util class. To benefit from this conversion also elsewhere not just on button elements.